app/javascript/src/libs/day.js

```diff
@@ -0,0 +1,6 @@
+import dayjs from "dayjs";
```

Comment:
Why have you named the folder as `libs` and why have you named the file as `day.js`?

Reply:
> Why have you named the folder as libs
I missed that in the slack message you've specified that the folder name is `lib`.
Every other folder in the same level has a plural name. That is why I've named it as `libs`. 
I'll correct it to `lib`.

> why have you named the file as day.js
I meant dayjs, but accidentally typed it as day.js. My bad. I'll correct it.